scripts/qa_crowbarsetup.sh

```diff
@@ -4116,8 +4116,17 @@ function oncontroller_testsetup
     fi
 
     # this file is created by the designate barclamp
-    designate_pools="/etc/desigate/pools.crowbar.yaml"
-    [[ -e $designate_pools ]] && designate-manage pool update --file $designate_pools
+    designate_pools="/etc/designate/pools.crowbar.yaml"
+    if [[ -e $designate_pools ]]; then
+        designate-manage pool update --file $designate_pools
+        # default values from the node.neutron.dns_domain
+        openstack tld create --name local
+        openstack zone create --email admin@example.org openstack.local.
```

Comment:
shouldn't this be done by the neutron integration of the barclamp?

Reply:
without pools, designate cannot create zones, (so they can be XFRd to be resolvable)